test/features/olm/subscription_operator.feature

```diff
@@ -0,0 +1,17 @@
+@disabled
+@olm
+@release
+Feature: Deploy Kogito Resources from subscription
+
+  Background:
+    Given Namespace is created
+
+  Scenario Outline: Deploy Kogito Operator from subscription
+    When Kogito Operator is deployed from subscription using channel "<channel>"
+    Then Kogito Operator should be installed from subscription with dependencies
```

Comment:
This step will check that the installed version of the operator matches with the CSV version, otherwise it fails.

Reply:
which CSV version is that ?

Reply:
The CSV version is checked from the subscription depending on the channel. It's using this method: https://github.com/kiegroup/kogito-cloud-operator/blob/c873d19970784825e8547f1367ee2649dfb2ab6f/pkg/infrastructure/operator.go#L61